Makefile

```diff
@@ -4,10 +4,17 @@ CLOUD9_DIR:=/var/lib/cloud9
 JEKYLL:=$(shell which jekyll)
 HTMLPROOFER:=$(shell which htmlproofer)
 NPM:=$(shell which npm)
+MRAA:=$(shell which mraa-gpio)
+EXDIR:= ./examples/extras/C-modulesFromJS/MRAA_Examples/
 
 all: ./bone101
 
 ./bone101: Makefile _config.yml
+ifdef MRAA
```

Comment:
Don't do a build here for on-target examples. This build is for the bone101 HTML content.